build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -82,7 +124,7 @@
     />
   </Target>
   <Target Name="_InstallBcl"
-      Inputs="$(_MonoPath)\mcs\class\lib\monodroid\mscorlib.dll"
+      Inputs="@(_BclProfileItems)"
       Outputs="$(OutputPath)lib\xbuild-frameworks\MonoAndroid\v1.0\mscorlib.dll;$(OutputPath)lib\xbuild-frameworks\MonoAndroid\v1.0\RedistList\FrameworkList.xml">
```

Comment:
If we change/expand `@(_BclProfileItems)`, we can list them all as outputs here too:

    <ItemGroup>
      <_BclAssembly Include="mscorlib.dll" />
      <_BclAssembly Include="System.dll" />
      ...
    </ItemGroup>
    <ItemGroup>
      <_BclProfileItems Include="@(_BclAssembly->'$(_MonoPath)\mcs\class\lib\monodroid\%(Identity)')" />
    </ItemGroup>
    <ItemGroup>
      <_BclInstalledItem Include=@(_BclAssembly->'$(OutputPath)lib\xbuild-frameworks\MonoAndroid\v1.0\%(Identity)')" />
    </ItemGroup>

Then you can use `@(_BclInstalledItem)` as an Output of `_InstallBcl`, with minimal duplication.